subsys/net/ip/tcp.c

```diff
@@ -2994,6 +3039,56 @@ uint16_t net_tcp_get_supported_mss(const struct tcp *conn)
 	return 0;
 }
 
+int net_tcp_set_option(struct net_context *context,
+			   enum tcp_conn_option option,
+			   const void *value, size_t len)
```

Comment:
Please align the arguments to the opening brackets.

Reply:
Fixed